lib/ansible/modules/network/aci/aci_bd_dhcp_label.py

```diff
@@ -0,0 +1,171 @@
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_bd_dhcp_label
+short_description: Manage DHCP Label on Cisco ACI fabrics (dhcp:Lbl)
+description:
+- Manage DHCP Label on Cisco ACI fabrics (dhcp:Lbl)
+- More information from the internal APIC class
+  I(dhcp:Lbl) at U(https://pubhub-prod.s3.amazonaws.com/media/apic-mim-ref/docs/MO-dhcpLbl.html).
+author:
+- sig9 (sig9@sig9.org)
+version_added: '2.5'
+requirements:
+- ACI Fabric 1.0(3f)+
+notes:
+- A DHCP relay label contains a name for the label, the scope, and a DHCP option policy.
+  The scope is the owner of the relay server and the DHCP option policy supplies DHCP clients
+  with configuration parameters such as domain, nameserver, and subnet router addresses.
+options:
+  bd:
+    description:
+    - The name of the Bridge Domain.
+    required: yes
+  description:
+    description:
+    - Specifies a description of the policy definition.
+    required: no
+  dhcp_label:
+    description:
+    - The name of the DHCP Relay Label.
+    required: yes
+  dhcp_option:
+    description:
+    - The DHCP option is used to supply DHCP clients with configuration parameters
+      such as a domain, name server, subnet, and network address.
+    required: no
+  owner:
+    description:
+    - Represents the target relay servers ownership.
+    required: yes
+    choices: [ infra, tenant ]
+    default: infra
+  tenant:
+    description:
+    - The name of the Tenant.
+    aliases: [ tenant_name ]
+    required: yes
+'''
+
+EXAMPLES = r'''
+- name: Create a new DHCP Relay Label to a Bridge Domain
+  aci_bd_dhcp_label:
+    host: "{{ apic }}"
+    username: "{{ username }}"
+    password: "{{ password }}"
+    validate_certs: false
+    tenant: "{{ tenant }}"
+    bd: "{{ inventory_hostname }}"
+    dhcp_label: "{{ dhcp_label }}"
+    owner: "{{ owner }}"
+    state: "present"
+
+- name: Remove a DHCP Relay Label for a Bridge Domain
+  aci_bd_dhcp_label:
+    host: "{{ apic }}"
+    username: "{{ username }}"
+    password: "{{ password }}"
+    validate_certs: false
+    tenant: "{{ tenant }}"
+    bd: "{{ inventory_hostname }}"
+    dhcp_label: "{{ dhcp_label }}"
+    owner: "{{ owner }}"
+    state: "absent"
+
+- name: Query a DHCP Relay Label of a Bridge Domain
+  aci_bd_dhcp_label:
+    host: "{{ apic }}"
+    username: "{{ username }}"
+    password: "{{ password }}"
+    validate_certs: false
+    tenant: "{{ tenant }}"
+    bd: "{{ inventory_hostname }}"
+    dhcp_label: "{{ dhcp_label }}"
+    owner: "{{ owner }}"
+    state: "query"
+'''
+
+RETURN = r'''
+#
+'''
+
+from ansible.module_utils.aci import ACIModule, aci_argument_spec
+from ansible.module_utils.basic import AnsibleModule
+
+
+def main():
+    argument_spec = aci_argument_spec
```

Comment:
`aci_argument_spec` is now a function and needs to be called like: `argument_spec = aci_argument_spec()`